src/main/java/org/folio/marc4ld/service/marc2ld/field/FieldMapperImpl.java

```diff
@@ -55,7 +55,6 @@ public Collection<Resource> createResources(Resource parent) {
   private Resource setProperties(Resource resource) {
     var existProp = mapperHelper.getProperties(resource);
     var properties = fieldRule.mergeProperties(dataField, controlFields, existProp);
-    labelService.setLabel(resource, properties);
```

Comment:
Label was computed before setting all properties of resource. Hence, removed it from here.